clients/js/test/getAssets.test.ts

```
@@ -30,36 +30,35 @@ DAS_API_ENDPOINTS.forEach((endpoint) => {
     t.deepEqual(compressedAsset!.interface, 'V1_NFT');
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)